keras/backend/mxnet_backend.py

```diff
@@ -292,7 +309,7 @@ def __abs__(self):
     def __add__(self, other):
         if isinstance(other, KerasSymbol):
             return KerasSymbol(
-                mx.sym.broadcast_add(
+                mx.sym.elemwise_add(
```

Comment:
Nice job. Thanks for identifying this bug.

Reply:
Why is it not a broadcast add? w * x + b, I think b can be a broadcasted?